config/jasmine.json

```diff
@@ -8,6 +8,7 @@
   ],
   "helpers": [
     "node/**/*-helper*.js",
-    "any/**/*-helper*.js"
+    "any/**/*-helper*.js",
+    "../node_modules/foam2/test/helpers/testcontext.js"
```

Comment:
That's right. I believe Jasmine will `JSON.parse()` this file, whereas Karma loads configuration files by `require()`ing them as Node JS modules.